erpnext_taxjar/hooks.py

```diff
@@ -16,19 +16,19 @@
 
 
 doc_events = {
-	"Quotation" : {
-		"validate": "erpnext_taxjar.api.set_sales_tax"
-	},
```

Comment:
Question... won't this remove taxes from quotations on the backend if they start one manually?

Do we know if this is a problem?